### PR TITLE
start 5.4.7 changelog

### PR DESCRIPTION
**Pull request by ashnwade**
This PR addresses no issue. 
Starting the changelog for v5.4.7.

### CODE REVIEW

File: changelog/5.4.7.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* Improved shard restoration logic. 
+* Improved logging around Scheduled Search retries.
+* Improved logging for indexer startup and shutdown.
+
+
+## Ingesters
+
+### Bug Fixes
+
+* Fixed an issue where the Attach directive was missing some entries.
```
Comment (john-floren-gravwell) [inline changelog/5.4.7.md:46]: This only happened if the entries hit the cache, we might mention that

File: changelog/5.4.7.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+### Additions
+
+* Added a new option to the [HTTP Flow node](/flows/nodes/http) to allow interpretation of `Content-Type` and response casting.
+* Added optional keys to the [Throttle Flow node](/flows/nodes/throttle) so that users can throttle based on a value in an Alert. 
+* Added the ability for non-admin users to mass delete Alerts that they own.
+* Added icons for Alerts to improve sharing visibility.
+* Added icons for Alerts to improve visibility when associating with Scheduled Searches and Flows. 
+* Added repair for indexer storage header that failed.
+* Allowed duplicated structure data in syslog.  
+* Updated charts to sort by magnitude.
```
Comment (michael-wisely-gravwell) [inline changelog/5.4.7.md:16]:
```suggestion
* Introduced new option to sort certain charts by field or magnitude.
```

**Review by michael-wisely-gravwell — CHANGES_REQUESTED** [review]
Just one small suggestion!

**Review by michael-wisely-gravwell — APPROVED** [review]
LGTM :rocket:

**Pull request merged**